tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -184,14 +184,26 @@ you may need to hard reload and clear your browser cache.
 
 If `package:devtools_app_shared`, `package:devtools_extensions`, or
 `package:devtools_shared` have unreleased changes, these packages may need to be
-published to pub.
+published to pub. Publish the backages whith versions they have, even if it is
+`.dev` versions.
```

Comment:
let's remove this. This release was more of a one-off note since I upgraded the minimum SDK versions for these packages to pre-release versions. When you have prerelease versions for the dart and flutter sdk mins, you will get a warning when you publish to publish the package as a pre-release version.

Reply:
So, do we want to instruct not to publish if version is .dev? Or ask in team chat?

Reply:
There is already a comment below to ask in team chat for context around publishing these packages.

Reply:
So if team said yes and versions are dev, they are still ok to publish, right? Or we want it to be said explicitly in chat? 
I want to give instructions, because dev versions look alerting and create extra mental effort for the release process.

Reply:
After thinking about this a bit more, perhaps we should have different instructions for the devtools packages that are shipped on pub. They should follow their own release cadence and each have their own considerations. For example, because package:dds depends on package:devtools_shared, we have to be careful about timing devtools_shared publishes with DevTools rolls into google3 and into the Dart SDK.

So my recommendation is to leave this part as is for now, and follow up with better docs for publishing the DevTools pub packages. Adding the commands is a nice addition though.